### PR TITLE
Release 4.3 fix feedback

### PR DESCRIPTION
**Pull request by wivwiv**

### CODE REVIEW

File: zh_CN/getting-started/config.md [hunk 1]
```diff
@@ -174,4 +174,10 @@ listener.tcp.example = 12345
 
 ## 配置更新
 
-配置项会在 EMQX Broker 与扩展插件被启动时读取并载入，EMQX Broker 目前尚不支持运行时更新配置，但由于扩展插件支持动态加载和卸载，因此可以在修改插件配置后重新加载插件来应用最新的配置项。
\ No newline at end of file
+{% emqxce %}
+配置项会在 EMQX 与扩展插件启动时读取并载入，EMQX 开源版尚不支持运行时更新配置（企业版支持这一能力），但由于扩展插件支持动态加载和卸载，因此可以在修改插件配置后重新加载插件来应用最新的配置。
```
Comment (LenaLenaPan) [inline zh_CN/getting-started/config.md:178]:
```suggestion
EMQX 开源版尚不支持在运行时更新配置，如有相关需求，可以考虑迁移至 EMQX 企业版。但由于配置项会在 EMQX 与扩展插件启动时读取并载入，因此您可在修改插件配置后重新加载插件来应用最新的配置。
```

File: zh_CN/getting-started/config.md [hunk 2]
```diff
@@ -174,4 +174,10 @@ listener.tcp.example = 12345
...
+{% endemqxce %}
+
+{% emqxee %}
+配置项会在 EMQX 与扩展插件启动时读取并载入，启动之后可以通过 Dashboard 与 REST API 更新部分配置，除此之外还可以在修改插件配置后重新加载插件来应用最新的配置。
```
Comment (LenaLenaPan) [inline zh_CN/getting-started/config.md:182]:
```suggestion
 EMQX 会在启动或插件启动时读取并载入配置项信息。后续如希望修改配置，可通过 Dashboard 或 REST API 在运行时更新配置，或在修改插件配置后重新加载插件来应用最新的配置。
```

File: en_US/rule/backend_tdengine.md [hunk 3]
```diff
@@ -5,12 +5,14 @@
 TaosData(https://www.taosdata.com)
 Enables efficient, real-time data ingestion, processing and monitoring of TB and even PB scale data per day, generated by billions of sensors and data collectors. TDengine can be widely applied to IoT, Industrial Internet, Connected Vehicles, DevOps, Energy , Finance and many other use-cases.
 
-EMQX supports saving data to TDengine via **Send to Web Server**, and also provides native TDengine drivers on EMQX Enterprise for direct saving.
+Install TDengine by  [Huawei Cloud](https://marketplace.huaweicloud.com/product/OFFI454488918838128640) or Docker:
```
Comment (LenaLenaPan) [inline en_US/rule/backend_tdengine.md:8]:
```suggestion
Deploy TDengine by  [Huawei Cloud](https://marketplace.huaweicloud.com/product/OFFI454488918838128640) or Docker:
```

File: en_US/getting-started/config.md [hunk 4]
```diff
@@ -151,3 +151,13 @@ Of course, in this case, we recommend that you copy the default Listener configu
...
 
 At this time, there are three available values for each of our configuration items, which are the global value, the value set in Zone and the default value, and their priority order is: Zone> Global> Default.
+
+## Apply Configuration
+
+{% emqxce %}
+Configuration items are loaded when EMQX starts or when the plugin starts. EMQX Community Edition does not yet support runtime configuration updates (Enterprise Edition supports this capability), but since the plugin supports dynamic loading and unloading, it is possible to reload the plugin to apply the latest configuration after modifying the plugin configuration.
```
Comment (LenaLenaPan) [inline en_US/getting-started/config.md:158]:
```suggestion
The EMQX open source edition does not support updating the configuration at runtime. If there is a need, you can consider migrating to EMQX Enterprise. To apply the latest settings, you can reload extension plugins as EMQX reads and loads configuration items during startup. 
```

File: en_US/getting-started/config.md [hunk 5]
```diff
@@ -151,3 +151,13 @@ Of course, in this case, we recommend that you copy the default Listener configu
...
+{% endemqxce %}
+
+{% emqxee %}
+Configuration items are loaded when EMQX starts or when the plugin starts. some configurations can be updated via Dashboard and REST API, and the latest configuration can be applied by reload the plugin after modifying the plugin configuration.
```
Comment (LenaLenaPan) [inline en_US/getting-started/config.md:162]:
```suggestion
EMQX will read and load the configuration items during startup or when extension plugins are started. You can update the configuration at runtime through the Dashboard or REST API, or reload the plug-in after modifying the plug-in configuration to apply the latest configuration.
```

**Comment by wivwiv** [comment]
Branches starting with the `release-` prefix are probably protected and I can't apply your suggestion, so pr is closed. 

https://github.com/emqx/emqx-docs/pull/1634

**Pull request closed without merging**